chaincode-labs/chaincode-seminars/lightning-network/2019-07-22-rene-pickhardt-onion-routing-with-htlcs.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Rene Pickhardt - Onion Routing with HTLCs on the Lightning Network 
+Transcript_by: Dulce Villarreal
+categories: ['chaincode-seminars']
```

Comment:
This category doesn't help much imo and also chaincode seminars resources might change in the future. I would agree with the categorization if maybe that's an indicator that this is quality material, but hopefully everything in here are considered quality resources. Tags are much better in categorization of the material.